jobs/payment-jobs/tasks/ejv_partner_distribution_task.py

```diff
@@ -140,8 +141,15 @@ def get_disbursement_by_distribution_for_partner(partner):
             )
         # ################################################################# END OF Legacy way of handling disbursements.
         # Partner disbursements - New
-        # Partial refunds need to be added to here later, although they should be fairly rare as most of them are from
         # NRO (NRO is internal, meaning no disbursement needed.)
+        disbursement_rows, distribution_code_totals = EjvPartnerDistributionTask._add_partner_disbursements(
+            partner, disbursement_date, disbursement_rows, distribution_code_totals)
+
+        return disbursement_rows, distribution_code_totals
+
+    @staticmethod
```

Comment:
add separate function _add_partner_disbursements to solve:
poetry run pylint tasks
************* Module tasks.ejv_partner_distribution_task
tasks/ejv_partner_distribution_task.py:63:4: R0914: Too many local variables (16/15) (too-many-locals)